# cam0 is not used, but expected from the code
cam0:
  T_cam_imu:
  - [0.999975, 0.007038, -0.000812, -0.037983]
  - [-0.007035, 0.999969, 0.003395, 0.002513]
  - [0.000836, -0.003389, 0.999994, -0.001260]
  - [0.000000, 0.000000, 0.000000, 1.000000]
  cam_overlap: [1]
  camera_model: pinhole
  distortion_coeffs: [0.008701, -0.018283, 0.038756, -0.025935]
  distortion_model: equidistant
  intrinsics: [469.650280, 469.551357, 375.629402, 236.808264]
  resolution: [752, 480]
  rostopic: /camera/rgb/image_raw
rgb:
  T_rgb_cam0:
  - [0.999968, 0.003327, -0.007251, -0.001814]
  - [-0.003260, 0.999953, 0.009178, -0.028674]
  - [0.007281, -0.009154, 0.999932, -0.020372]
  - [0.000000, 0.000000, 0.000000, 1.000000]
  distortion_coeffs: [0.000000, 0.000000, 0.000000, 0.000000, 0.000000]
  distortion_model: plumb_bob
  intrinsics: [616.57562255859385, 616.5757446289062, 311.1305236816406, 245.28565979003906]
  resolution: [640, 480]
depth:
  distortion_coeffs: [0.000000, 0.000000, 0.000000, 0.000000, 0.000000]
  distortion_model: plumb_bob
  depth_correction: [-0.025925, 0.024734]
  intrinsics: [475.37042236328125, 475.37030029296875, 307.33660888671875, 245.8182373046875]
  resolution: [640, 480]
vicon:
  T_vicon_cam:
  
  # good one:
#  - [-0.220723885829879,  -0.973264408569225,   0.063540201735239,  -0.007794527818806]
#  - [0.973655234243020,  -0.223698692292081,  -0.044208369094355,  -0.007306909182233]
#  - [0.057240292236576,   0.052108406991665,   0.996999630323634,  -0.004633578760994]
#  - [0.0,0.0,0.0,1.0]
  - [-0.215431815897845,-0.973520729506582,0.0764625524026632,0.0792926739000586]
  - [0.973323891433095,-0.220396510126072,-0.0637650428509632,-0.0419912198240765]
  - [0.0789286707381670,0.0606858100812801,0.995031405228139,-0.0534376764302762]
  - [0.0,0.0,0.0,1.0]
